packages/visual-editing/src/ui/OverlayDragPreview.tsx

```diff
@@ -58,22 +58,30 @@ const Root = styled.div<{
   }
 `
 
-export const OverlayDragPreview: FunctionComponent<{skeleton: DragSkeleton}> = ({skeleton}) => {
-  const minSkeletonWidth = 100
-  const minSkeletonHeight = 100
+function clamp(number: number, min: number, max: number): number {
+  return number < min ? min : number > max ? max : number
+}
+
+function map(number: number, inMin: number, inMax: number, outMin: number, outMax: number): number {
+  const mapped: number = ((number - inMin) * (outMax - outMin)) / (inMax - inMin) + outMin
+  return clamp(mapped, outMin, outMax)
+}
 
+export const OverlayDragPreview: FunctionComponent<{skeleton: DragSkeleton}> = ({skeleton}) => {
   const maxSkeletonWidth = window.innerWidth / 2
   const scaleFactor = skeleton.w > maxSkeletonWidth ? maxSkeletonWidth / skeleton.w : 1
 
-  const showSkeleton =
-    skeleton.w * scaleFactor >= minSkeletonWidth && skeleton.h * scaleFactor >= minSkeletonHeight
-
   const offsetX = skeleton.offsetX * scaleFactor
   const offsetY = skeleton.offsetY * scaleFactor
 
   const prefersDark = usePrefersDark()
   const theme = useTheme_v2()
 
+  const radius = theme.radius[~~map(skeleton.w, 0, 1920, 1, theme.radius.length - 2)]
```

Comment:
it would be great to be able to do this in CSS alone... it would also be cool if sanity UI had some kind of `auto` radius property for cards 🤔

Reply:
For sure!